CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
```

Comment:
Can we just get rid of this changelog?

Reply:
yeah 👍🏼 :I think we should use the github releases to document notable changes between versions after published.